src/gallery/slider.js

```diff
@@ -328,14 +313,25 @@ const onShowFn = () => {
 	const items = sliderContainer.querySelectorAll( `.${prefixClassname}-item` )
 	const nextButton = sliderContainer.querySelector( '.next' )
 	const previousButton = sliderContainer.querySelector( '.previous' )
+	let tapped = false
 
 	renderNext( 0 )
 	applyGestureEvent()
 
 	sliderContainer.addEventListener( 'click', ( e ) => {
 		if ( e.target.className === `${prefixClassname}-item` ||
 				e.target.tagName === 'IMG' ) {
-			toggleFocusMode()
+			if ( !tapped ) {
+				tapped = setTimeout( () => {
+					tapped = null
+					toggleFocusMode()
+				}, 300 )
```

Comment:
previously there wasn't timeout 300 for the single tap toggling, is there any way to not timeout it? but still keep the behaviour of double tap doesn't toggle it?

or, should we keep the caption visible when in zoom mode?

Reply:
I tried originally with https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event but it doesn't support mobile unfortunately. After that [a search pointed me](https://stackoverflow.com/questions/24058241/touch-device-single-and-double-tap-events-handler-jquery-javascript) to this idea using timeouts... so would have to think about this a bit more if we wanted to not use timeouts. Are you concerned that using a timeout here might make thing more complicated down the road? 

> or, should we keep the caption visible (hidden?) when in zoom mode?

We can confirm with Sudhanshu. Last time I talked with him he did mention the behavior he is expecting is that a single tap can toggle the focus mode regardless of whether the image is zoomed in or not

Reply:
> single tap can toggle the focus mode regardless of whether the image is zoomed in or not

Thanks. Then I think there is nothing we can do if we use one click event to detect either single or double tap without timeout. I'm closing this thread conversation now unless others comment on this or have better idea.